src/towncrier/build.py

```diff
@@ -260,7 +260,7 @@ def __main(
             "What is seen below is what would be written.\n",
             err=to_err,
         )
-        click.echo(content)
+        click.echo(content.rstrip() + "\n")
```

Comment:
Just a minor comment ... and just asking :)

why strip the content here ?

I think it's ok to have one extra newline, but leave the other newlines as they are defined in the template.

This can help debug / preview the actual fragment.

Reply:
I was on the fence about this one, but I think you're right, it makes more sense for debugging to keep the whitespace as-is.

I'll need to adjust quite a few tests though :)

Reply:
My understanding is that with this change, you can't configure some newlines in the template, as they are stripped :) ... and this is contrary to the purpose of this PR.

Reply:
But we can leave it like this, if this will cause a big change.

Reply:
This particular change is only for the draft command though ;)

So the newlines are preserved when generating the actual file, but stripped for the "preview".

I do, however, like the debugging argument, that's worth adjusting the tests.